package.json

```diff
@@ -58,6 +58,11 @@
       "type": "object",
       "title": "Dash Configuration",
       "properties": {
+        "dash.zeal": {
+          "type": "boolean",
+          "default": false,
+          "description": "Specify if you're using Zeal because it handles the Dash URI differently."
+        },
```

Comment:
I feel that users will be more happy if this plugin can automatically work with zeal without they need to setup this setting value